src/project-tests/scatter-plot-tests.js

```diff
@@ -1,146 +1,287 @@
-                });
-            });
-
-
-            it('10. I can see multiple tick labels on the x-axis that show the year.', function() {
-                const xAxisTickLabels = document.querySelectorAll("#x-axis .tick");
-                FCC_Global.assert.isAbove(xAxisTickLabels.length, 0, "Could not find tick labels on the x axis ")
-                xAxisTickLabels.forEach(label => {
-                    //match check if this is a year
-                    FCC_Global.assert.match(label.textContent, /[1-2][0-9][0-9][0-9]/, 'X-axis tick labels do not show the year ');
-                });
-            });
-
-            it('11. I can see that the range of the x-axis labels are within the range of the actual x-axis data.', function() {
-                const xAxisTickLabels = document.querySelectorAll("#x-axis .tick");
-                const MIN_YEAR = 1994;
-                const MAX_YEAR = 2016;
-                FCC_Global.assert.isAbove(xAxisTickLabels.length, 0, "Could not find tick labels on the x axis ")
-                xAxisTickLabels.forEach(label => {
-                    FCC_Global.assert.isAtLeast(label.textContent, MIN_YEAR, "x axis labels are below the range of the actual data ");
-                    FCC_Global.assert.isAtMost(label.textContent, MAX_YEAR, "x axis labels are above the range of the actual data ");
-                });
-            });
-
-            it('12. I can see that the range of the y-axis labels are within the range of the actual y-axis data.', function() {
-                const yAxisTickLabels = document.querySelectorAll("#y-axis .tick");
-                const MIN_TIME = new Date(0, 0, 0, 0, MIN_MINUTES, 0, 0);
-                const MAX_TIME = new Date(0, 0, 0, 0, MAX_MINUTES, 0, 0);
-                FCC_Global.assert.isAbove(yAxisTickLabels.length, 0, "Could not find tick labels on the y axis ");
-                yAxisTickLabels.forEach(label => {
-                    var timeArr = label.textContent.split(":");
-                    var mins = timeArr[0];
-                    var secs = timeArr[1];
-                    var date = new Date(0, 0, 0, 0, mins, secs, 0);
-                    FCC_Global.assert.isAtLeast(date, MIN_TIME, "y axis labels are below the range of the actual data ");
-                    FCC_Global.assert.isAtMost(date, MAX_TIME, "y axis labels are above the range of the actual data ");
-                });
-            });
-
-            it('13. I can see a legend that has id="legend".', function() {
-                FCC_Global.assert.isNotNull(document.getElementById('legend'), 'There should be an element with id="legend" ');
-            });
-        });
-        
-        testToolTip(document.querySelectorAll('.dot'), "data-year", "data-xvalue")
+    describe('#Content', function() {
+      it('1. I can see a title element that has a corresponding id="title".',
```

Comment:
@tbushman These reformatting changes should be put in a separate PR. Otherwise it is hard to tell which changes are related to your new feature, and which are only related to re-formatting. 

It does make sense to correctly format the tests you changed. Which means this code will have inconsistent formatting. But that should be very temporary since as soon as we merge this the next step would be for you to submit another PR fixing the formatting for this entire file.

If that doesn't make sense, please feel free to reach out to me on Skype and we can talk about it.